docs/source/changelog.rst

```diff
@@ -1,7 +1,7 @@
 Changelog
 =========
 
-2021.12.0
+v2021.12.0
 ---------
```

Comment:
Currently we're getting warnings like 

```
/home/docs/checkouts/readthedocs.org/user_builds/dask/checkouts/8534/docs/source/changelog.rst:5: WARNING: Title underline too short.

v2021.12.0
---------
/home/docs/checkouts/readthedocs.org/user_builds/dask/checkouts/8534/docs/source/changelog.rst:55: WARNING: Title underline too short.

v2021.11.2
---------
```

in the documentation build logs telling us we need to add an extra `-` along with the `v`

```suggestion
----------
```

Reply:
> From [sphinx-doc/sphinx#1449](https://github.com/sphinx-doc/sphinx/issues/1449) the issue seems to be with docutils which requires id to begin with a letter. See https://repo.or.cz/docutils.git/blob/HEAD:/docutils/docutils/nodes.py#l2241

@jrbourbeau the id must start with a letter, if we don't want `v` we can write version headings like `Version 2021.12.0`. Pandas follows this convention, see https://pandas.pydata.org/docs/whatsnew/index.html#version-1-3

Reply:
I think it might also be possible to specify an explicit target (although I haven't tested it). Something like:

```
.. _v2021.12.0:
2021.12.0
---------
```

I feel like there must be a link somewhere in the Sphinx docs, but I can't find it. Instead, I found this in the (related) MyST docs: https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets

Reply:
Thanks @bryanwweber that worked.